modules/es7.system.global.js

```diff
@@ -1,4 +0,0 @@
-// https://github.com/tc39/proposal-global
-var $export = require('./_export');
-
-$export($export.S, 'System', { global: require('./_global') });
```

Comment:
You might want to hold off on removing `System.global` until tc39/proposal-global#20 is cleared up.

Reply:
@aciccarello maybe. I'm watching this issue, but `global` is already on stage 3.